xbmc/settings/AdvancedSettings.cpp

```diff
@@ -289,8 +289,8 @@ void CAdvancedSettings::Initialize()
   m_iVideoLibraryRecentlyAddedItems = 25;
   m_bVideoLibraryCleanOnUpdate = false;
   m_bVideoLibraryUseFastHash = true;
-  m_bVideoLibraryImportWatchedState = false;
-  m_bVideoLibraryImportResumePoint = false;
+  m_bVideoLibraryImportWatchedState = true;
```

Comment:
```suggestion
  m_bVideoLibraryImportWatchedState{true};
```

See: https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#114-default-member-initialization

Reply:
done